### PR TITLE
Update to Plex ver 1.19.1.2589

### PR DESCRIPTION
**Pull request by TBK**
https://forums.plex.tv/t/plex-media-server/30447/328

### CODE REVIEW

**Comment by frebib** [comment]
Hum, it looks like they changed something. This is going to need more digging
https://drone.spritsail.io/spritsail/plex-media-server/396/1/2

If you can't/don't manage to fix it, I'll have a look tomorrow

**Comment by frebib** [comment]
These are the differences between the versions
https://gist.github.com/frebib/403f07cfb44ad4054341fa91861cb887/raw

**Comment by frebib** [comment]
OpenSSL will also need to be updated
```
frebib@frebib-PC /t/t/u/l/p/lib> strings libcrypto.so.1.1| grep "^OpenSSL 1\.1.*" 
OpenSSL 1.1.1d  10 Sep 2019
```
and curl, possibly
```
frebib@frebib-PC /t/t/u/l/p/lib> strings libcurl.so | grep -Ei '^curl-[0-9]'
curl-7.68.0-2cfac302f/lib/vtls/openssl.c
```

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-ARG PLEX_VER=1.18.9.2578-513b381af
-ARG PLEX_SHA=8854a6aebe9ac375431b2e3e5addc7b7421dea4a
+ARG PLEX_VER=1.19.1.2589-2de7f3266
+ARG PLEX_SHA=2b7a08c41b99b40a3cd45cf687ffd3de3c61c60c
 ARG XMLSTAR_VER=1.6.1
 ARG CURL_VER=curl-7_66_0
```
Comment (frebib) [inline Dockerfile]: Can you bump this to 7.68 please

**Review by frebib — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 ARG ZLIB_VER=1.2.11
-ARG OPENSSL_VER=1.0.2u
+ARG OPENSSL_VER=1.1.1d
```
Comment (frebib) [inline Dockerfile]: Looking upstream, 1.1.1e is available. If this works, we should probably pull it in. It looks like it contains only bugfixes
https://www.openssl.org/news/changelog.html#openssl-111

File: Dockerfile (hunk 1 above)
Comment (TBK) [reply Dockerfile]: You do not want 7.69.1? https://curl.haxx.se/changes.html#7_69_1

**Comment by frebib** [comment]
```
frebib@frebib-PC ~/n/p/s/plex-media-server> openssl s_client -host plex.nerdhouse.io -port 32400 -status -showcerts -verify 1 < /dev/null 2>/dev/null | grep TLS     
New, TLSv1.3, Cipher is TLS_AES_128_GCM_SHA256
```
Nice

File: Dockerfile (hunk 1 above)
Comment (frebib) [reply Dockerfile]: Yeah sure, I'd be willing to try it. Last time we bumped curl though, it didn't end well. There was random breakage with Plex being able to request stuff

**Comment by TBK** [comment]
> ```
> frebib@frebib-PC ~/n/p/s/plex-media-server> openssl s_client -host plex.nerdhouse.io -port 32400 -status -showcerts -verify 1 < /dev/null 2>/dev/null | grep TLS     
> New, TLSv1.3, Cipher is TLS_AES_128_GCM_SHA256
> ```
> 
> Nice

That is nice.

File: Dockerfile (hunk 1 above)
Comment (TBK) [reply Dockerfile]: https://abi-laboratory.pro/index.php?view=timeline&l=curl

Comment (frebib) [reply Dockerfile]: Awesome, that's good by me then. It was likely an ABI incompatibility, given that we were replacing libs without relinking

**Comment by frebib** [comment]
lgtm, thanks for persevering!

**Comment by TBK** [comment]
Thanks for the assistance :+1:

**Pull request merged**